seed/tests/test_account_views.py

```diff
@@ -324,36 +318,26 @@ def test_remove_user_from_org_missing_org_id(self):
         self.org.add_member(u)
         with self.assertRaises(NoReverseMatch):
             self.client.delete(
-                reverse_lazy('api:v2:organizations-remove-user') + '?organization_id=' + str(
+                reverse_lazy('api:v3:organization-users-remove') + '?organization_id=' + str(
                     self.org.id),
-                data=json.dumps({'user_id': u.id}),
-                content_type='application/json',
             )
 
     def test_remove_user_from_org_missing_user_id(self):
         u = User.objects.create(username='b@b.com', email='b@be.com')
         self.org.add_member(u)
 
-        resp = self.client.delete(
-            reverse_lazy('api:v2:organizations-remove-user', args=[self.org.id]),
-            content_type='application/json',
-        )
-        self.assertDictEqual(
-            json.loads(resp.content),
-            {
-                'status': 'error',
-                'message': 'missing the user_id'
-            })
+        with self.assertRaises(NoReverseMatch):
+            resp = self.client.delete(
+                reverse_lazy('api:v3:organization-users-remove', args=[self.org.id]),
+            )
```

Comment:
What do you think about removing these two tests entirely? My thought is that since we use the nested_router package to implement `/api/v3/organizations/{org_id}/users/{user_id}`, we don't necessarily need to test what happens when org_id or user_id isn't included.

Reply:
Per discussion, I'll go ahead and remove them